strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/maven/MavenArtifactControllerTest.java

```diff
@@ -840,14 +840,18 @@ public void testMetadataAtGroupAndArtifactIdLevel(@MavenRepository(repositoryId
 
         MavenArtifactDeployer artifactDeployer = buildArtifactDeployer(Paths.get(System.getProperty("java.io.tmpdir")));
 
+        try{
         // When
         artifactDeployer.generateAndDeployArtifact(pluginArtifact1, storageId, repositoryId);
         artifactDeployer.generateAndDeployArtifact(pluginArtifact2, storageId, repositoryId);
         artifactDeployer.generateAndDeployArtifact(pluginArtifact3, storageId, repositoryId);
         artifactDeployer.generateAndDeployArtifact(pluginArtifact4, storageId, repositoryId);
         artifactDeployer.generateAndDeployArtifact(artifact5, storageId, repositoryId);
         artifactDeployer.generateAndDeployArtifact(artifact6, storageId, repositoryId);
-
+        } catch (Exception e) {
+            logger.error("Error while generate And deploy artifacts", e);
```

Comment:
Cosmetic: Braces on new lines please. :)
```suggestion
        }
        catch (Exception e)
        {
            logger.error("Error while generate And deploy artifacts", e);
```